app/logic/searchUsers.py

```diff
@@ -15,6 +15,8 @@ def searchUsers(query):
         for participant in results:
             if participant not in resultsDict:
                 resultsDict[f"{participant.firstName} {participant.lastName} ({participant.username})"] = f"{participant.firstName} {participant.lastName} ({participant.username})"
+                if phoneNumber=="true":
```

Comment:
Why are we using the string "true" and not a boolean?

Reply:
Also, the variable should have a different name that indicates what it controls. A variable called `phoneNumber` should just contain a phone number.